VTUL/iawa
# LIBTD-1666: Hyrax 2.4.1 Migration

**JIRA Ticket**: https://webapps.es.vt.edu/jira/browse/LIBTD-1666

# What does this Pull Request do?
This PR migrates the IAWA application from Hyrax 2.1.0 to 2.4.1. We don't need to do this upgrade. However, the Hyrax [release notes](https://github.com/samvera/hyrax/releases) indicate that this upgrade includes security fixes (e.g., cross-site scripting vulnerabilities) as well as accessibility updates. 

# What are the changes?
* Modified the `Gemfile` to use `hyrax 2.4.1`.
* Ran `bundle update hyrax` to update the `Gemfile.lock` file.
* Updated related `browse-everything` config and views to accommodate the reverted version.
* Updated README.md to reflect the hyrax version

# How should this be tested?

Test in both local `development` and `production` environments.
* Create a single work and confirm that it is created properly.
* Create a single collection and confirm that it is created properly.
* Run multiple batch uploads and confirm that they were created properly.
* Modify existing controlled vocabulary and confirm that they are modified properly.
* In development mode, run `bundle exec rspec` as the `hydra` user and ensure that no tests fail.

# Additional Notes:
* What branch to be used for testing this PR? `LIBTD-1666`

# Interested parties
@tingtingjh
